### ISSUE TITLE
[Accessibility][Keyboard navigation]: Keyboard focus indicator is not properly visible on 'Bar' button in forward navigation.

### ISSUE THREAD

**Issue by jdhuntington** [OfficeDev/office-ui-fabric-react#10584]
Tracking all of the issues brought up with the 12/18 accessibility pass on Pivot control 
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B07b9e428-09cf-4a02-8fa1-05015209994a%7D&action=default&uid=%7B07B9E428-09CF-4A02-8FA1-05015209994A%7D&ListItemId=1635&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble) 

This issue is tracking a single entry from #7434

**Comment by jdhuntington** [comment]
Unable to repro. Default styling has drastically changed since the creation of this issue, and the new look and feel seems to have addressed.
